sdk/automanage/azure-resourcemanager-automanage/CHANGELOG.md

```diff
@@ -1,14 +1,141 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2024-10-10)
 
-### Features Added
+- Azure Resource Manager Automanage client library for Java. This package contains Microsoft Azure SDK for Automanage Management SDK. Automanage Client. Package tag package-2022-05. For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
 
 ### Breaking Changes
 
-### Bugs Fixed
+#### `models.Report` was modified
```

Comment:
This is due to flatten being removed by service: https://github.com/Azure/azure-rest-api-specs/pull/21381/files